libsrc/common/ctype.s

```diff
@@ -0,0 +1,92 @@
+; ctype.s
+;
+; This file is part of
+; cc65 - a freeware C compiler for 6502 based systems
+;
+; https://cc65.github.io
+;
+; See "LICENSE" file for legal information.
+;
+; Character specification table, matching serveral consoles.
```

Comment:
as far as I know the 'c' in ctype stands for 'character', not for 'console'. So I'm wondering where the term 'console' comes from. In case you refer to the game consoles supported by cc65 I'd prefer 'machines' or 'targets'.

Reply:
Contrary to the ctype_common.inc (also machines/targets), this source file fits mainly to game consoles (yes, and the Apple2).